lib/better_html/better_erb.rb

```diff
@@ -52,10 +52,19 @@ def generate(template, source)
         klass ||= self.class.erb_implementation
 
         escape = self.class.escape_ignore_list.include?(template.type)
-        generator = klass.new(
-          erb,
+
+        options = {
           escape: escape,
           trim: (self.class.erb_trim_mode == "-")
+        }
+        if ActionView::Base.annotate_rendered_view_with_filenames && template.format == :html
```

Comment:
We should not put a Rails configuration check directly in the library code. Let's add a new config for this gem and set the config in the `Railtie`